src/app/reporting/reporting.c

```diff
@@ -67,6 +68,24 @@ EmberEventControl emberAfPluginReportingTickEventControl;
 
 EmAfPluginReportVolatileData emAfPluginReportVolatileData[REPORT_TABLE_SIZE];
 
+/** @brief Configured
+ *
+ * This callback is called by the Reporting plugin whenever a reporting entry
+ * is configured, including when entries are deleted or updated. The
+ * application can use this callback for scheduling readings or measurements
+ * based on the minimum and maximum reporting interval for the entry. The
+ * application should return EMBER_ZCL_STATUS_SUCCESS if it can support the
+ * configuration or an error status otherwise. Note: attribute reporting is
+ * required for many clusters and attributes, so rejecting a reporting
+ * configuration may violate ZigBee specifications.
+ *
+ * @param entry   Ver.: always
+ */
+EmberAfStatus emberAfPluginReportingConfiguredCallback(const EmberAfPluginReportingEntry * entry)
```

Comment:
Ported from Silabs SDK

Reply:
Again, should say that in the commit message.

Please have on of the silabs folks sign off on these bits license-wise?

Reply:
@wehale @selissia - is it ok to port this from the Silabs SDK?